MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -263,6 +290,19 @@ - (void)monitorAVPlayerItem {
                       forKeyPath:@"status"
                          options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
                          context:MUXSDKAVPlayerItemStatusObservationContext];
+        
+        if([_player.currentItem.asset isKindOfClass:[AVURLAsset class]]) {
+            AVURLAsset *asset = (AVURLAsset *) _player.currentItem.asset;
+            NSLog(@"MUXSDK-INFO - Loading HLS master manifest from asset URL: %@", asset.URL);
+            __weak MUXSDKPlayerBinding *weakSelf = self;
```

Comment:
`__weak` here so we don't accidentally retain `self` causing a memory leak. If the player binding has been deallocated by the time the completion block is called, nothing will happen